src/pages/Account/Index.tsx

```diff
@@ -10,6 +10,8 @@ import {useGetAccount} from "../../api/hooks/useGetAccount";
 import LoadingModal from "../../components/LoadingModal";
 import Error from "./Error";
 import {AptosNamesBanner} from "./Components/AptosNamesBanner";
+import {useGlobalState} from "../../global-config/GlobalConfig";
+import {NetworkName} from "@aptos-labs/wallet-adapter-react";
```

Comment:
nit: can use sdk for that https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/index.ts#L18

https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/utils/api-endpoints.ts#L13